src/data/level3-rest-links.json

```diff
@@ -0,0 +1,155 @@
+{
+  "https://level3.rest/patterns/list#list": {
+    "rel": "list",
```

Comment:
These `rel` properties should probably just go. They're not used, an an artifact from an earlier conversion script. If we'd ever want a 'short form' to appear, I'd probably give it a name like 'label' and not re-use rel to avoid confusion.

Reply:
Heh heh I think that's about the time we started chatting about what they were good for! I pushed  change to remove them.

Reply:
Thnx =) gonna do an update tonight!